packages/feedback/src/core/createMainStyles.ts

```diff
@@ -38,41 +39,30 @@ function getThemedCssVariables(theme: FeedbackTheme): string {
 /**
  * Creates <style> element for widget actor (button that opens the dialog)
  */
-export function createMainStyles(
-  d: Document,
-  colorScheme: 'system' | 'dark' | 'light',
-  themes: FeedbackThemes,
-): HTMLStyleElement {
-  const style = d.createElement('style');
+export function createMainStyles(colorScheme: 'system' | 'dark' | 'light', themes: FeedbackThemes): HTMLStyleElement {
+  const style = DOCUMENT.createElement('style');
   style.textContent = `
 :host {
   --bottom: 1rem;
-  --right: 1rem;
+  --right: 1rem; /* this is the font-size of the page, not the shadowroot */
```

Comment:
Is this comment in the right place?

Reply:
let's remove this or make it a JS comment, otherwise it may just add unnecessary bytes I guess?